modelon/impact/client/entities/model.py

```diff
@@ -276,3 +281,112 @@ def new_experiment_definition(
+                include_patterns and exclude_patterns. Example: If
+                top_level_inputs = 'my_inputs*', then all input variables matching the
+                pattern 'my_inputs*' will be generated as inputs, and all other inputs
+                not matching the pattern as model variables. If top_level_inputs = '',
+                then no input is imported as an input.
+                Default value: None (which means all inputs are kept as inputs)
+                Type: str or a list of strings
+            step_size:
+                Specify what value to set for the parameter for step size in the
+                generated model. By default the parameter is set to zero, which
+                inturn means that the step size will be set during simulation based
+                on simulation properties such as the time interval.
+                This can also be manually set to any real non-negative number.
+                The value of the step size parameter can also be set via the function
+                set_step_size, which must be invoked before importing the model.
+                Default value: 0.0 (which during simulation is set according to the
+                description above).
+
+        Example::
+
+            workspace = client.get_workspace("test")
+            package = workspace.get_model('MyPackage')
+            fmu_content = package.import_fmu('C:/A.fmu').wait()
+            fmu_content = package.import_fmu('C:/B.fmu',
+                class_name="MyPackage.Model").wait()
+
+        """
+        resp = self._sal.project.project_get(
+            self._project_id, vcs_info=False, size_info=False
+        )
+        project = Project(
+            resp["id"],
+            resp["definition"],
+            resp["projectType"],
+            None,
+            self._sal,
+        )
+        modelica_content = project.get_modelica_library_by_name(self._class_name)
+        if not modelica_content:
+            raise ValueError(
+                f"No modelica package with name {self._class_name} exists "
+                f"in the Project - '{project.name}'"
+            )
+        class_name = class_name or ".".join(
+            [
+                self._class_name,
+                os.path.split(fmu_path)[-1].strip('.fmu'),
+            ]
+        )
+        resp = self._sal.project.fmu_import(
+            self._project_id,
+            modelica_content.id,
+            fmu_path,
+            class_name,
+            overwrite,
+            include_patterns,
+            exclude_patterns,
+            top_level_inputs,
+            step_size=step_size,
+        )
+
+        return FMUImportOperation[Model](
+            resp['data']['location'],
+            self._workspace_id,
+            self._project_id,
+            self._sal,
+            Model.from_operation,
+        )
+
+    @classmethod
+    def from_operation(cls, operation: BaseOperation[Model], **kwargs: Any) -> Model:
```

Comment:
Should this method be private?

Reply:
Yes I think it does. I will push another PR doing this for all the from_operation class methods. Thanks! :)